documentation/modules/exploit/linux/http/cisco_rv340_lan.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This module exploits two vulnerabilities, an authentication bypass (CVE-2022-20705) and a command injection vulnerability (CVE-2022-20707),
+to execute code on Cisco RV340x Small Business Routers as the `www-data` user. The command injection occurs in the `upload.cgi` script,
```

Comment:
In your main exploit you say this only affects RV340 and RV345 routers yet here you say its affects RV340 routers only. Looks like a disgrepancy.

Reply:
Apologies for discrepancies.  I have only tested against my own RV340 router. But I agree the information should be in the module.